### ISSUE TITLE
Can't add GLControl to .NET framework WinForm window

### ISSUE THREAD

**Issue by AllocatorMK** [opentk/opentk#1115]
### Description
When trying to add a GLControl to a WinForm form, the control fails to load.

### Repro steps

1. Create a basic .NET framework WinForms C# project.

2. Create a form.

3. Install OpenTK and OpenTK.GLControl NUGET packages.

4. From the toolbox, drag the GLControl item (under OpenTK) onto the form. This shows the error message.

### Expected behavior

When dragging the item to the form, it should add the control to the form, and the control should be visible.

### Actual behavior

When dragging the item to the form, an error message pops up saying "Failed to load toolbox item "GLControl". It will be removed from the toolbox."

### Related information 

* Windows 10
* OpenGL v4.6.0 NVIDIA 461.77
* OpenTK v3.2.0 and OpenTK.GLControl v3.1.0 (NUGET packages)
* Project uses .NET frameworks 4.6.1

**Comment by jvbsl** [comment]
try creating the GLControl in code and not with the designer

**Comment by Shadowblitz16** [comment]
this might be a workaround but this should be considered a bug 
it should be supported by the designer as its easier to use.

**Comment by jvbsl** [comment]
It is a workaround only if it works, and that is the first question. I don't even know if it does work by doing it in code I only have anecdotal evidence for that. That's why we need to analyze the problem first. Apart from that don't forget that it was decided, that updates to otk 3.2 come through PRs and you are very welcome to do so, apart from that most effort of our limited time is spent in the new version.
That is why I would classify it as a inconvenience and the fix as an enhancement, as it is still usable and not really breaking.
What I would do(if that workaround does work) is use a placholder panel and make the GLControl child of that and Dock=Full(in Code), then you can design all you want without any problem.

TL:DR PRs to fix that are welcome ;)

**Comment by varon** [comment]
GLControl is largely obsolete at the moment, and needs a 4.x version to be created, which should hopefully fix this.
